web_resources/FME_exporter.js

```diff
@@ -236,16 +236,20 @@ $(function () {
 
             $pledge.find('.without-images .item .title').each((i, elBonus) => {
                 const bonus = $(elBonus).text().trim();
-                const insuranceRegexResult = /(\d+)\s+Month\s+Insurance/i.exec(bonus);
+                const insuranceRegexResult = /(\d+)(\s+|-)Month(s|)\s+Insurance/i.exec(bonus);
                 let insurance = null;
                 if (insuranceRegexResult !== null && insuranceRegexResult[1]) {
                     insurance = parseInt(insuranceRegexResult[1]);
                 }
                 const lti = /Lifetime\s+Insurance/i.test(bonus);
+                const iae = /IAE\s+Insurance/i.test(bonus);
 
                 if (lti) {
                     lifetimeInsurance = true;
                 }
+                if (iae) {
+                    insurance = 120;
```

Comment:
I would prefer directly to send IAE flag in the payload. ;)

Reply:
Fine for me. Will change it. I wanted it do work with FM immediately yesterday ;-)

Reply:
Question: Atm you can set insurance months AND lti AND iae, what is actually not possible. Wouldn't it be a good idea to, instead, introduce a insurance_type variable set to "lti" OR "iae" OR, lets say, "monthly"? I don't want to alter your code without asking in such a way ;-)

Reply:
Question: Atm you can set insurance months AND lti AND iae, what is actually not possible. Wouldn't it be a good idea to, instead, introduce a insurance_type variable set to "lti" OR "iae" OR, lets say, "monthly"? (Plus insurance_months for the number) I don't want to alter your code without asking in such a way ;-)

Reply:
Good idea, `insuranceType: enum<"lti"|"iae"|"monthly">` + `insuranceDuration`. :)

Reply:
I made the broke changes. I would suggest to keep the properties Nullable for edfe cases ;-)